CMakeLists.txt

```diff
@@ -204,7 +221,9 @@ list(APPEND EXPORT_MODULES
     "cmake/AwsSIMD.cmake"
     )
 
-install(FILES ${EXPORT_MODULES} DESTINATION "lib/cmake/")
+install(FILES ${EXPORT_MODULES}
+        DESTINATION "${LIBRARY_DIRECTORY}/cmake/${PROJECT_NAME}"
```

Comment:
This should be CMAKE_PROJECT_NAME, no?

Reply:
Had to look this up:

https://stackoverflow.com/questions/38938315/difference-between-cmake-project-name-and-project-name

In this situation, there's only one project command so the two would be equivalent. I don't really think there would be much reason to add another `project()` command, so I'm happy to use whichever one.

Reply:
Aye, ok. I guess let's just error on the side of being consistent. We use CMAKE_PROJECT_NAME everywhere, let's keep doing it :)